astropy/io/votable/tests/test_vo.py

```diff
@@ -178,20 +178,23 @@ def _test_regression(tmp_path, _python_based=False, binary_mode=1):
 
 
 @pytest.mark.xfail("legacy_float_repr")
+@pytest.mark.usefixtures("without_legacy_printoptions")
```

Comment:
Could we just do it for the whole test file?

Reply:
In principle yes but there doesn't seem to be a need for it: tests that fail without legacy print options are actually pretty rare in the code base so I don't expect to find more when we remove the global setting, which we should be able to do as soon as we merge #15065